kratos/solving_strategies/builder_and_solvers/builder_and_solver.h

```diff
@@ -504,16 +504,12 @@ class BuilderAndSolver
      */
     virtual void Clear()
     {
-        this->mDofSet.clear(); // = DofsArrayType();
-
-        TSparseSpace::Clear(this->mpReactionsVector);
-        // 			this->mReactionsVector = TSystemVectorType();
-
-        if (this->GetEchoLevel() > 0)
-        {
+        this->mDofSet.clear();
```

Comment:
I think this clear doesn't do the same as the others. Could you check that ?

Reply:
Seems to me like it is not resized to zero (this happens in the `space`)

We could also do `this->mDofSet.DofsArrayType();` then I guess it would have size zero

Is this what you mean?

Reply:
ping @RiccardoRossi

Reply:
the "old" way was to assign it to an empty object

     this->mDofSet = DofsArrayType()

that surely destroys it...

Reply:
so using `clear` is ok? Or shoudl I also resize it to zero?

Reply:
Ping @RiccardoRossi